src/App.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div id="app">
+    <div id="nav">
+      <router-link to="/">Home</router-link> |
+      <router-link to="/about">About</router-link> |
```

Comment:
Let's keep just 2 `router-link`s, /home and /todo-list. Load your todo list list on the /todo-list page